hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -217,6 +234,13 @@ public class RocksDBCheckpointDiffer implements AutoCloseable,
         TimeUnit.MILLISECONDS);
     this.suspended = new AtomicBoolean(false);
 
+    this.stripedSSTLock = SimpleStriped.readWriteLock(
```

Comment:
Let us not do add a new configuration for locks, reuse OzoneManagerLock

Reply:
See reply above.